init/jupyter/gravitino-flink-hive-example.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
This case looks good to me. Maybe we can add more operations like alter table,drop table later.

Reply:
I tested the operations, and `DROP` works as expected. However, I encountered some errors when trying to use `ALTER`

Py4JJavaError: An error occurred while calling o31.executeSql.
: java.lang.NoClassDefFoundError: org/apache/gravitino/shaded/org/apache/commons/compress/utils/Lists

I find it quite strange because other commands work fine. I’m wondering if the issue could be related to the `gravitino-flink-connector-runtime`. Any insights or suggestions would be appreciated.

Reply:
@TungYuChiang Could you replace the `import org.apache.commons.compress.utils.Lists;` to `import com.datastrato.gravitino.shaded.com.google.common.collect.Lists;` in https://github.com/apache/gravitino/blob/main/flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/catalog/BaseCatalog.java#L32 and rebuild it. Then try it again. Thanks.

Reply:
@TungYuChiang I create an issue https://github.com/apache/gravitino/issues/5534. Could you submit a patch to fix it?

Reply:
@coolderli  Thank you for your guidance! 
I have already left a comment under #3354.